### PR TITLE
[Modules] Updated Network ExpressRouteCircuits to new dependencies approach

### PR DESCRIPTION
**Pull request by MrMCake**
# Description

- Updated Network ExpressRouteCircuits to new dependencies approach

| Pipeline |
| - |
| [Network: ExpressRouteCircuits](https://github.com/Azure/ResourceModules/actions/workflows/ms.network.expressroutecircuits.yml) |

# Type of Change

Please delete options that are not relevant.

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Update to documentation

### CODE REVIEW

File: modules/Microsoft.Network/expressRouteCircuits/.test/common/deploy.test.bicep [hunk 1]
```diff
@@ -0,0 +1,78 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for a testing purposes')
```
Comment (MrMCake) [inline modules/Microsoft.Network/expressRouteCircuits/.test/common/deploy.test.bicep:6]:
```suggestion
@description('Optional. The name of the resource group to deploy for testing purposes')
```

File: modules/Microsoft.Network/expressRouteCircuits/.test/min/deploy.test.bicep [hunk 2]
```diff
@@ -0,0 +1,40 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for a testing purposes')
```
Comment (MrMCake) [inline modules/Microsoft.Network/expressRouteCircuits/.test/min/deploy.test.bicep:6]:
```suggestion
@description('Optional. The name of the resource group to deploy for testing purposes')
```

File: modules/Microsoft.Network/expressRouteCircuits/.test/common/deploy.test.bicep [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'ms.network.expressroutecircuits-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'nercom'
```
Comment (eriqua) [inline modules/Microsoft.Network/expressRouteCircuits/.test/common/deploy.test.bicep:14]:
```suggestion
param serviceShort string = 'nerccom'
```

**Review by eriqua — APPROVED** [review]
:ok

**Pull request merged**